src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/TestDriverComponent.scala

```diff
@@ -62,9 +62,14 @@ trait TestDriverComponent extends DriverComponent with DataAccess {
       case ref: Entity => Workflow(UUID.randomUUID.toString, WorkflowStatuses.Submitted, testDate, Option(AttributeEntityReference(ref.entityType, ref.name)), inputResolutions(ref))
     })
 
+    val failedWorkflows = (workflowEntities collect {
```

Comment:
This is gonna make a success and a failure for each entity, no? That doesn't make sense (and how come it doesn't trigger your exception [up here](https://github.com/broadinstitute/rawls/pull/404/files#diff-556cde67db5479e77c934fcdc9e66814R137)?)